geocoding_ios/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 2.1.1
+
+* Removes deprecation warnings.
```

Comment:
I think it is even better if you list the concrete change instead of the result of the change.

```suggestion
* Removes obsolete version check in `toPlacemarkDictionary`. This removes [insert deprecation warning instance] from occurring.
```